data/Nigeria/Senate/sources/instructions.json

```diff
@@ -0,0 +1,26 @@
+{
+  "sources": [
+    {
+      "file": "morph/data.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "everypolitician-scrapers/nigeria-national-assembly",
+        "query": "SELECT * FROM data WHERE js_position = 'Sen' AND area != ''"
```

Comment:
sorry for not spotting this earlier, but this gives a build warning as it doesn't include an `ORDER BY` (as it otherwise creates lots of spurious diffs when the source order changes later)